src/Host/Client/Impl/Session/RSession.cs

```diff
@@ -83,7 +85,7 @@ internal sealed class RSession : IRSession, IRCallbacks {
             CanceledBeginInteractionTask = TaskUtilities.CreateCanceled<IRSessionInteraction>(new RHostDisconnectedException());
         }
 
-        public RSession(int id, string name, IBrokerClient brokerClient, IExclusiveReaderLock initializationLock, Action onDispose) {
+        public RSession(int id, string name, IBrokerClient brokerClient, IExclusiveReaderLock initializationLock, Action onDispose, IServiceContainer services) {
```

Comment:
removed the service requirement.

---
In reply to: [111820325](https://github.com/Microsoft/RTVS/pull/3419#discussion_r111820325) [](ancestors = 111820325)